action/concrete_factory.go

```diff
@@ -51,7 +57,7 @@ func NewConcreteFactory(softLayerClient sl.Client, options ConcreteFactoryOption
 			"configure_networks": NewConfigureNetworks(vmFinder),
 
 			// Disk management
-			"create_disk": NewCreateDisk(nil),
+			"create_disk": NewCreateDisk(diskCreator),
 			"delete_disk": NewDeleteDisk(nil),
```

Comment:
The delete disk(and attach disk/detach disk/find disk) will be submitted in the following PRs